src/form/form-text-input.tsx

```diff
@@ -7,18 +7,21 @@ import { ErrorMessage } from './error-message/error-message';
 interface Props {
   name: string;
   multiline?: boolean;
+  isAllowCleanButton?: boolean;
 }
 
-export const FormTextInput: FC<Props> = ({ name, multiline = false }) => {
+export const FormTextInput: FC<Props> = ({ name, multiline = false, isAllowCleanButton }) => {
   const [field, meta] = useField<string>(name);
 
   return (
     <>
       <StyledTextInput
         multiline={multiline}
         value={field.value}
+        isError={!!meta.error}
```

Comment:
I think `isError` should be more complicated.
Take a look at `ErrorMessage` component `hasError` vaiable